### DISCUSSION TITLE
How change the logo with PasswordPusher hosting

### DISCUSSION THREAD

**Discussion by Zaisuru** [pglombardo/PasswordPusher#1910] [Q&A] [answered]
Hi,

I need help with my problem.
I installed PwPush on my Debian 12 virtual machine and now I would like to change the logo to put the
 my company logo.
But I have no idea how to handle it

Can you help me please ?

**Comment by pglombardo** [comment, marked as answer]
Hi @Zaisuru - the documentation is here: https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#rebranding

The whole document covers how you can configure Password Pusher to your needs.  Let me know if you have any questions or issues.  I'd be happy to help out.

**Comment by Zaisuru** [comment]
Hi,

Thank you for your reply .
So, if I understand correctly for my solution (on site), I just need to change the logo
`light_logo: 'logo-transparent-sm-bare.png'
  # dark_logo: 'logo-transparent-sm-dark-bare.png'`

and I need to replace it in the folder with my logo?

**Reply by pglombardo** [reply]
Not really - `logo-transparent-sm-bar.png` is the default logo.

If you want to set your own logo, place your logo images locally into /var/lib/pwpush/logos/. You would then mount that directory into the container:

`docker run -d -p "5100:5100" -v /var/lib/pwpush/logos:/opt/PasswordPusher/public/logos pglombardo/pwpush:release`

then set your LOGO environment variables (or settings.yml options) to:

`PWP__BRAND__LIGHT_LOGO=/logos/mylogo.png`

Or alternatively you could just set an environment variable with a fully qualified remote path:

`PWP__BRAND__LIGHT_LOGO =https://pwpush.fra1.cdn.digitaloceanspaces.com/dev%2Facme-logo.jpg`

**Comment by Zaisuru** [comment]
I don't understand
But on my server I placed the logo in /var/lib/ and I created the pwpush and logos folder
After that I placed the logo in it
And in settings.yml I don't have a line for the logo
![image](https://github.com/pglombardo/PasswordPusher/assets/32138397/ca61f12a-62f2-4df5-8e8e-3f3c220096b3)

**Comment by Zaisuru** [comment]
litlle up  ?

**Comment by Zaisuru** [comment]
in You procedure talk only with the command docker run

But for me i don't have docker because in have installing the  solution on my server with the git clone and this step
![image](https://github.com/pglombardo/PasswordPusher/assets/32138397/79889057-7743-4848-b43a-aed57370017f)

**Reply by pglombardo** [reply]
Hi @Zaisuru - ah ok.  I assumed Docker since that is the most common deployment method.  Did you resolve the issues with the logo?

**Reply by Zaisuru** [reply]
No, now i test a new deployment with docker
